site/source/docs/building_from_source/index.rst

```diff
@@ -34,10 +34,14 @@ For example, something like this can work:
 
       mkdir build
       cd build/
-      cmake ../llvm -DCMAKE_BUILD_TYPE=Release -DLLVM_ENABLE_PROJECTS='lld;clang' -DLLVM_TARGETS_TO_BUILD="host;WebAssembly" -DLLVM_INCLUDE_EXAMPLES=OFF -DLLVM_INCLUDE_TESTS=OFF
+      cmake ../llvm -DCMAKE_BUILD_TYPE=Release -DLLVM_ENABLE_PROJECTS='lld;clang' -DLLVM_TARGETS_TO_BUILD="host;WebAssembly" -DLLVM_INCLUDE_EXAMPLES=OFF -DLLVM_INCLUDE_TESTS=OFF -DLLVM_ENABLE_ASSERTIONS=ON
```

Comment:
How about making assertions a comment afterwards? I think in general most users wouldn't want that overhead.

(In the emsdk itself we do it on normal builds but not on LTO builds, which are what we ship in most releases, iirc - @dschuff is that right?)

Reply:
Done!